packages/apps/transfer/src/components/Global/WalletConnectButton/index.tsx

```diff
@@ -6,9 +6,9 @@ import React, { FC } from 'react';
 export const WalletConnectButton: FC = () => {
   const { t } = useTranslation();
   return (
-    <Button.Root title={t('Connect your wallet')} color="positive">
+    <Button title={t('Connect your wallet')} color="positive">
       {t('Connect your wallet')}
       <SystemIcon.Link />
```

Comment:
should this be updated to use the prop?

Reply:
Yes, to all of those haha I was too focused on just fixing the errors. Will update them.